assembl/utils.py

```diff
@@ -13,15 +13,19 @@ def format_date(datetime_to_format):
     return datetime_to_format.strftime('%d/%m/%Y')
 
 
-def get_posts(idea, start=None, end=None, include_deleted=None):
+def get_posts(idea, start=None, end=None,
+              include_deleted=None,
+              publication_states=None,
```

Comment:
So, given that in a lot of the export funcs you pass a default `PublicationStates.published`, perhaps you should make this state the default here. Also, update the docstrings below to reflect the new parameters that you've added.

Reply:
as None value is available, i don't like having a non None default value
i fix the doc